### PR TITLE
Feature 118

### PR DESCRIPTION
**Pull request by JustinaMary**

### CODE REVIEW

**Review by peterpolman — APPROVED** [review]

File: migrate-mongo-config.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
+// In this file you can configure migrate-mongo
+
+const config = {
+  mongodb: {
+    // TODO Change (or review) the url to your MongoDB:
+    url: "mongodb://root:root@127.0.0.1:27017/admin",
+
+    // TODO Change this to your database name:
+    databaseName: "admin",
```
Comment (peterpolman) [inline migrate-mongo-config.js]: @JustinaMary I guess that for this we need to introduce a second environment variable MONGO_DB containing the database name which is environment specific.

Comment (peterpolman) [reply migrate-mongo-config.js]: Also add this variable to the .env.example and .env.test files

File: migrate-mongo-config.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+  },
+
+  // The migrations dir, can be an relative or absolute path. Only edit this when really necessary.
+  migrationsDir: "migrations",
+
+  // The mongodb collection where the applied changes are stored. Only edit this when really necessary.
+  changelogCollectionName: "changelog",
+
+  // The file extension to create migrations and search for in migration dir 
+  migrationFileExtension: ".js",
```
Comment (peterpolman) [inline migrate-mongo-config.js]: @JustinaMary Can we use .ts files easily with this setting for database migrations? I would prefer that.

File: README.MD [hunk 3]
```diff
@@ -121,3 +124,49 @@ Swagger Documentation is provided in the controllers of the routes and lives at
 -   [Local API Specs](https://localhost:3000/v1/docs/)
 -   [Staging API Specs](https://dev.api.thx.network/v1/docs/)
 -   [Production API Specs](https://api.thx.network/v1/docs/)
+
+## 8. Database Schema Migration
+
+### 8.1 To add a new migration script, use the following CLI command
+
+```
+migrate-mongo create name-of-my-script
```
Comment (peterpolman) [inline README.MD:133]: @JustinaMary Please also add `migrate-mongo` as a dev dependency to the package.json.

Comment (JustinaMary) [reply README.MD:133]: Sure

File: migrate-mongo-config.js [hunk 4]
```diff
@@ -0,0 +1,34 @@
+// In this file you can configure migrate-mongo
+
+const config = {
+  mongodb: {
+    // TODO Change (or review) the url to your MongoDB:
+    url: "mongodb://root:root@127.0.0.1:27017/admin",
```
[the remark this answers is not in the document]
Comment (JustinaMary) [reply migrate-mongo-config.js]: Completed

File: migrate-mongo-config.js (hunk 1 above)
Comment (JustinaMary) [reply migrate-mongo-config.js]: Done

File: migrate-mongo-config.js (hunk 2 above)
Comment (JustinaMary) [reply migrate-mongo-config.js]: Done

**Pull request merged**